### ISSUE TITLE
Updating mod information with a mod that no longer exists (I.E. deleted) causes NullReference

### ISSUE THREAD

**Issue by EpicestGamer** [FelisDiligens/Fallout76-QuickConfiguration#7]
I have a mod installed called Fallout 3 Fanfare ( https://www.nexusmods.com/fallout76/mods/1011 ), it's been deleted from the nexus. When 76 Quick Config runs across it while updating mod data it crashes, when you'd expect it to just skip over the link.

```
Operating system:  Windows 10 64-bit
Program version:   1.9.4
User agent:        Fo76QuickConfiguration/1.9.4 (Windows NT 10.0.19042.0; x64) .NETFramework/4.7.2
Running as admin:  No
Game edition:      SteamSystem locale:     en-US
App locale:        en-US
************** Stack trace **************
If any files are listed (like "D:\Workspace\...\*.cs:line 123"):
Those are files on *my* computer, so don't worry if you can't find them.

System.NullReferenceException: Object reference not set to an instance of an object.
   at Fo76ini.NexusAPI.NMMod.RequestInformation() in D:\Workspace\Fallout 76 Quick Configuration\Fallout76-QuickConfiguration\Fo76ini\NexusAPI\NMMod.cs:line 92
   at Fo76ini.NexusAPI.NexusMods.RequestModInformation(String url) in D:\Workspace\Fallout 76 Quick Configuration\Fallout76-QuickConfiguration\Fo76ini\NexusAPI\NexusMods.cs:line 127
   at Fo76ini.FormMods.UpdateRemoteModInfo(Action`1 ProgressChanged) in D:\Workspace\Fallout 76 Quick Configuration\Fallout76-QuickConfiguration\Fo76ini\Forms\FormMods\FormMods.cs:line 1650
   at Fo76ini.FormMods.<UpdateRemoteModInfoThreaded>b__87_1() in D:\Workspace\Fallout 76 Quick Configuration\Fallout76-QuickConfiguration\Fo76ini\Forms\FormMods\FormMods.cs:line 1551
   at Fo76ini.FormMods.<>c__DisplayClass94_0.<RunThreaded>b__0() in D:\Workspace\Fallout 76 Quick Configuration\Fallout76-QuickConfiguration\Fo76ini\Forms\FormMods\FormMods.cs:line 1709
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
```

This can easily be worked around by a user by removing the problematic link.

**Comment by DigiGlitch** [comment]
ran into this issue as well, quiet a few people are removing their mods due to the changes nexus made, so there are quiet a few old mods that no longer have a page

**Comment by FelisDiligens** [comment]
Should be fixed in v1.9.5
